### ISSUE TITLE
Time-limited editing of thread title (as per issues)

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/cyclescape#895]
User feedback, which is very sensible and has been a long-running quasi-bug:

"Is there any chance that time limited editing not only of the issue title but the thread too could be introduced? (In case of typos or because the info is wrong.)"

As a secondary matter, it would also be sensible that if the issue and thread title are the same, changing the issue title changes the thread (as if the user had written the issue that way in the first place). I don't think the other way round should be done though.

**Comment by mvl22** [comment]
> Is this a suggestion/request to allow the thread creator to edit the thread title only or the whole thread?

Yes, sorry for the potentially unclear spec I gave.

The criteria here is for the creator of the thread to be able to change the thread title within 24 hours, and no other attribute (group owner, etc.).

Certainly in my role as a site user rather than with a sysadmin hat on, I've found myself wanting to adjust the thread title a bit to make it clearer, shortly after posting.
